ajax/libs/angular-parse/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "angular-parse",
+  "version": "0.2.2",
+  "description": "Angular wrapper for Parse.com JavaScript SDK.",
+  "license": "MIT",
+  "author": "<ivnivnch@gmail.com>",
+  "filename": "angular-parse.js",
+  "keywords": [
+    "angular",
+    "angularjs",
+    "parse",
+    "parsesdk"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ivnivnch/angular-parse.git"
+  },
+  "npmName": "angular-parse",
+  "npmFileMap": [
```

Comment:
@sashberd Maybe we can use git auto-update config to get the latest version.
What do you think?
Then we can manually add old versions ('0.2.0', '0.2.1') from npm.
Because I think many developers would like to use newest versions.
If CDNJS can host the latest version with the auto-update config, it will good for everyone.

Thank you.